docs/source/ko/_toctree.yml

```
@@ -721,6 +721,8 @@
       - local: in_translation
         title: (번역중) Pix2Struct
       - local: in_translation
+      - local: model_doc/qwen2_vl
+        title: Qwen2VL
```

I think you may have cut off the `-local: model_doc/sam` under here

```suggestion
        title: Qwen2VL
```